amaranth-chrome-ext/src/CalorieLabeller.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * An object which labels a dish name as low, average, or high-calorie.
+ */
+class CalorieLabeller {
+  /**
+   * Creates a CalorieLabeller object.
+   * @constructor
+   * @param {Map<string, number>} tokenizer A mapping from dish name tokens to
+   * unique integers
+   * @param {tf.LayersModel} model Tensorflow.js layers ML model
+   */
+  constructor(tokenizer, model) {
+    /** @private @const @type {Map<string, number>} */
```

Comment:
Are these required? They don't seem to add as much value as just comments.

Reply:
I find them useful as code editor type hints. I wouldn't use them everywhere, but these variables are relatively important, and [it is allowed, and encouraged for templated types like Map, by the styleguide](https://google.github.io/styleguide/jsguide.html#features-declare-types-as-needed).